contrib/pkg/deprovision/gcp.go

```diff
@@ -61,8 +64,12 @@ func (o *gcpOptions) Validate(cmd *cobra.Command) error {
 		log.Info("Region is required")
 		return fmt.Errorf("missing region")
 	}
-	credsFile := os.Getenv("GOOGLE_CREDENTIALS")
-	projectID, err := gcpclient.ProjectIDFromFile(credsFile)
+
+	creds, err := gcputils.GetCreds(o.credsFile)
```

Comment:
This is not sufficient. This will only change what is used for determining the project ID. The uninstaller will not respect the `--creds-file` flag.